lib/xenops_server.ml

```diff
@@ -1761,7 +1758,7 @@ and perform ?subtask ?result (op: operation) (t: Xenops_task.task_handle) : unit
 					Some Needs_unplug
 				end in
 			let operations_of_request = function
-				| Needs_unplug -> Some (Atomic(VBD_unplug (id, true)))
+				| Needs_unplug -> Some (Atomic(VUSB_unplug id))
```

Comment:
Why `VBD_uplug` has become `VUSB_unplug`?

Reply:
Oh, I see, this should have been `VUSB_unplug` all along

Reply:
Can you please move this change to its own separate commit?
Can you also test if this alone actually fixes the issue

Reply:
Yes, I will  separate the commmit. BTW, i tested, it will fix the issue.

Reply:
I think Taoyong means with both commits applied this issue is fixed.